exo-setup/src/app-setup.ls

```diff
@@ -37,7 +37,7 @@ class AppSetup extends EventEmitter
       async.map-series
     else
       async.map
-    operation setups, (-> &0.start &1), (err) ~>
+    async.map-series setups, (-> &0.start &1), (err) ~>
```

Comment:
I see. Please calculate the `operation` variable right before you use it. I.e. move line 40 above line 33.